src/index.ts

```diff
@@ -1,5 +1,5 @@
 import Client, { Event } from './client';
-import LRU from 'lru-cache';
+import * as LRU from 'lru-cache';
```

Comment:
The library uses both CommonJS and ESM exports, and the customer who raised the issue does not allow ESM imports explicitly in their typescript config, so I've used this syntax.

Reply:
That's weird. Typically people go the other way since TS _only_ allows the ESM format (though it can resolve both)

Reply:
Very weird... 
I used their ts.config file and ran `tsc myfile.ts` and it complained about that. With   `"esModuleInterop": true` it was fine

Reply:
Modules in Node are in a terrible place at the moment. That's where it's losing a lot of devs to things like Deno and Bun since they do a _much_ better job outa the box. ¯\_(ツ)_/¯. This is good though, I'd have imported as `* as LRU` anyway unless the package specifically called out that the default export was an object. Destructuring in an `import` _looks_ like normal destructuring, but it's actually very different.